policyengine_us/variables/gov/states/az/tax/income/credit/az_tax_140_line56_increased_excise_tax_credit.py

```diff
@@ -0,0 +1,68 @@
+# Questions:
+# 1. Doesn't mention widow?
+# 2. Should I remove "federal"?
+
+# Some details that don't include in the coding:
+# 1. valid SSN for employement
+# 2. prison issues
+# 3. You are not claimed as a dependent by any other taxpayer
+# 4. The credit cannot exceed $100 per household. Do not claim this credit if someone else in your household has already claimed $100 of the credit. If someone else in your household has claimed less than $100, you may claim the credit as long as all credit claims filed from your household do not exceed $100
+
+from policyengine_us.model_api import *
+
+
+class az_increased_excise_tax_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "AZ Totaling Payments and Refundable Credits - increased excise tax credit"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.AZ
+    reference = "https://azdor.gov/forms/individual/form-140-arizona-resident-personal-income-tax-booklet"  # From Year 2022 (Line 56)
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.az.tax.income.credits.increased_excise_tax_credit
+        line12 = tax_unit(
+            "adjusted_gross_income", period
+        )  # should I remove "federal"?
+        filing_status = tax_unit("filing_status", period)
+        status = filing_status.possible_values
+        max_income = select(
```

Comment:
```suggestion
        max_income = p.maximum[filing_status]
```
instead of the select statement